src/ensembl/src/header/launchbar/LaunchbarButton.tsx

```diff
@@ -29,11 +29,11 @@ const LaunchbarButton: FunctionComponent<LaunchbarButtonProps> = (
   const imageButton = (
     <ImageButton
       classNames={{
-        [Status.DEFAULT]: styles.launchbarButtonImage,
-        [Status.ACTIVE]: styles.launchbarButtonSelectedImage,
+        [Status.UNSELECTED]: styles.launchbarButtonImage,
+        [Status.SELECTED]: styles.launchbarButtonSelectedImage,
         [Status.DISABLED]: styles.launchbarButtonDisabledImage
       }}
-      buttonStatus={imageButtonStatus}
+      status={imageButtonStatus as ImageButtonStatus}
```

Comment:
Do we know why `as ImageButtonStatus` has to be specified here? `getImageButtonStatus` function returns values of type `ImageButtonStatus` anyway; why do we have to declare ImageButtonStatus here as well?